Makefile

```diff
@@ -84,6 +84,9 @@ clean: clean-build clean-pyc clean-test ## Remove all build, test, coverage and
 release: dist ## package and upload a release
 	twine upload dist/*
 
+root-url:
+	@docker-compose port bigchaindb 9984
```

Comment:
I wonder why you used `@docker-compose` here rather than `@$(DC)`

Reply:
oh! I forgot to change it back after testing something. Will fix it